lib/helpers.ex

```diff
@@ -0,0 +1,46 @@
+defmodule Helpers do
```

Comment:
Having the GTFS parsing as a top-level module makes sense, but this seems likely to conflict with another module from somewhere else. As it's currently only used by the GTFS parsing/state, maybe `lib/gtfs/helpers.ex`? You can use `alias GTFS.Helpers` in the relevant modules to avoid any other code changes.